### PR TITLE
Comment out build/login/push

### PR DESCRIPTION
**Pull request by thom-at-redhat**
# Description

Comment out Travis fleet_management build/login/push.  The file was validated using the travis configuration explorer

Fixes # (issue)

## Type of change

What is it?

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [ ] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I run `go fmt ./...` to check that my code is properly formatted
- [ ] I run `go vet ./...` to check that my code is free of common Go style mistakes

### CODE REVIEW

**Comment by InsightsDroid** [comment]
Test results are available in [Ibutsu](https://url.corp.redhat.com/ibutsu-runs). The test run IDs are:
0047f25c-b76f-4323-895f-3b5e5e9b111c
88340b77-c546-49b7-98b8-cb83c1ade076

**Review by mgold1234 — COMMENTED** [review]
/lgtm

File: .travis.yml [hunk 1]
```diff
@@ -13,33 +13,33 @@ branches:
...
+    #   - if [ "$TRAVIS_PULL_REQUEST" != false ]; then echo "LABEL quay.expires-after=2d" >> ./Dockerfile; fi
+    #   - sed -i 's|registry.access.redhat.com/ubi8/ubi|quay.io/centos/centos:stream8|' ./Dockerfile
+    #   - sed -i 's|.*ubi-micro-build.*ubi.repo||' ./Dockerfile
+    #   - sed -i "s|${FLEET_MANAGEMENT_ORG}/libfdo-data|${LIBFDO_IMAGE}|" ./Dockerfile
+    #   - docker build -t ${EDGE_API_IMAGE} -f ./Dockerfile .
+    #   - docker login quay.io -u ${FLEETER_BOT:-${FLEETER_BOT_PR}} -p ${FLEETER_PASS:-${FLEETER_PASS_PR}} >/dev/null 2>&1
+    #   - docker push ${EDGE_API_IMAGE}
     - name: "Static Code Analysis"
       script:
         - docker run --rm -w /edge-api -v $(pwd):/edge-api ${LIBFDO_IMAGE} /bin/sh -c "staticcheck -go 1.15 -f stylish ./..."
```
[the remark this answers is not in the document]
Comment (thom-at-redhat) [reply .travis.yml]: Per your suggestion, commented out tests involving container

**Review by loadtheaccumulator — APPROVED** [review]
/lgtm

**Pull request merged**